packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts

```diff
@@ -30,169 +30,176 @@ import ParameterError from '../common/errors/parameter';
 import LoggerProxy from '../common/logs/logger-proxy';
 import {SpaceIDDeprecatedError} from '../common/errors/webex-errors';
 
-const MeetingInfoUtil: any = {};
-const meetingInfoError =
-  'MeetingInfo is fetched with the meeting link, SIP URI, phone number, Hydra people ID, or a conversation URL.';
+/**
```

Comment:
The refactor of utilv2.ts is required as without this refactor an empty declaration file was being created.

For the refactor 
1. Create a class MeetingInfoUtil
2. All the method inside the class are static. No src code changes are made.